day_7/checker_board.js

```diff
@@ -0,0 +1,42 @@
+/*
+psuedocode:
+Create function that takes in two arguments, length (x), and width (y).
+For every row (with an even index) print the pattern of 'X_' for every other row print '_X'.
+The number of 'X_' or '_X' printed are determined by the value of 'y'.
+Divide the value of 'y' by two, since characters are two digits long.
+*/
+
+function checkerboard(x, y) {
+    var result = '';
+    for (var i = 0; i < x; i++){
+        for(var j = 0; j < y; j++){
```

Comment:
Nested for loops can get tricky, but for 2 dimensional problems they can be a good solution.